Shared/Samples/Style geometry types with symbols/StyleGeometryTypesWithSymbolsView.swift

```diff
@@ -0,0 +1,157 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct StyleGeometryTypesWithSymbolsView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// A Boolean value indicating whether the edit styles view is presented.
+    @State private var isEditStyles = false
+    
+    var body: some View {
+        MapView(map: model.map, graphicsOverlays: [model.graphicsOverlay])
+            .toolbar {
+                ToolbarItem(placement: .bottomBar) {
+                    editStylesButton
+                }
+            }
+    }
+    
+    /// The button for presenting the edit styles view.
+    private var editStylesButton: some View {
+        Button("Edit Styles") {
+            isEditStyles = true
+        }
+        .popover(isPresented: $isEditStyles) {
+            editStyles
+                .presentationDetents([.fraction(0.5)])
+                .frame(idealWidth: 320, idealHeight: 380)
+        }
+    }
+    
+    /// The view for editing the styles of the geometries.
+    private var editStyles: some View {
+        NavigationStack {
+            SymbolsEditor(model: model)
+                .navigationTitle("Edit Styles")
+                .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    ToolbarItem(placement: .confirmationAction) {
+                        Button("Done") {
+                            isEditStyles = false
```

Comment:
I didn't use dismiss action since this toolbar is on `StyleGeometryTypesWithSymbolsView` and not `SymbolsEditor`. I can move it there if you think that is better, though.